### PR TITLE
Implement Throughput

### PR DESCRIPTION
**Pull request by Suseika**
Implementation of feature request #10

### CODE REVIEW

**Comment by dmarkov** [comment]
@suseika Thanks for the pull request, let me find a reviewer..

**Comment by dmarkov** [comment]
@caarlos0 please review

File: src/main/java/com/jcabi/dynamo/Throughput.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+package com.jcabi.dynamo;
+
+import com.amazonaws.services.dynamodbv2.model.ProvisionedThroughput;
+import com.jcabi.aspects.Tv;
+import javax.validation.constraints.NotNull;
+
+/**
+ * Throughput of a table.
+ * @author Georgy Vlasov (wlasowegor@gmail.com)
+ * @version $Id$
```
Comment (caarlos0) [inline src/main/java/com/jcabi/dynamo/Throughput.java]: @Suseika missing `@since` tag

File: src/test/java/com/jcabi/dynamo/ThroughputTest.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+     * @throws Exception If something went wrong
+     */
+    @Test
+    public final void adjustsThroughput() throws Exception {
+        final Table table = Mockito.mock(Table.class);
+        final Region region = Mockito.mock(Region.class);
+        final AmazonDynamoDB aws = Mockito.mock(AmazonDynamoDB.class);
+        Mockito.when(table.region()).thenReturn(region);
+        Mockito.when(region.aws()).thenReturn(aws);
+        new Throughput(table).adjust();
```
Comment (caarlos0) [inline src/test/java/com/jcabi/dynamo/ThroughputTest.java]: @Suseika this test asserts nothing... let's verify if the `updateTable` method was called with expected parameters...

**Comment by caarlos0** [comment]
@Suseika couple of comments above

**Comment by Suseika** [comment]
@caarlos0 Fixed that, please see the result

File: src/test/java/com/jcabi/dynamo/ThroughputTest.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+import org.mockito.Mockito;
+
+/**
+ * Test case for {@link Throughput}.
+ * @author Georgy Vlasov (wlasowegor@gmail.com)
+ * @version $Id$
+ */
+public class ThroughputTest {
+    /**
+     * Adjusts throughput on a table.
```
Comment (caarlos0) [inline src/test/java/com/jcabi/dynamo/ThroughputTest.java]: @Suseika `Throughput can change throughput of a table.`

**Comment by caarlos0** [comment]
@Suseika one more comment, also, build is failing : 

> [ERROR] /src/test/java/com/jcabi/dynamo/ThroughputTest.java[53]: The String "Customers" appears 2 times in the file. (MultipleStringLiteralsCheck)

File: src/test/java/com/jcabi/dynamo/ThroughputTest.java [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+     * @throws Exception If something went wrong
+     */
+    @Test
+    public final void adjustsThroughput() throws Exception {
+        final Table table = Mockito.mock(Table.class);
+        final Region region = Mockito.mock(Region.class);
+        final AmazonDynamoDB aws = Mockito.mock(AmazonDynamoDB.class);
+        Mockito.when(table.region()).thenReturn(region);
+        Mockito.when(table.name()).thenReturn("Customers");
```
Comment (caarlos0) [inline src/test/java/com/jcabi/dynamo/ThroughputTest.java]: @Suseika let's create a variable for this "Customers" literal

**Comment by Suseika** [comment]
@caarlos0 Done

**Comment by caarlos0** [comment]
@rultor merge

**Comment by caarlos0** [comment]
@Suseika thanks!

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by dmarkov** [comment]
@caarlos0 Thanks for your contribution, **19 mins** was added to your account, payment ID is `AP-1E029596U7392132E`, 47 hours and 40 mins spent total. extra minutes for review comments (c=4). +19 added to your rating, at the moment it is: [+2221](http://www.netbout.com/b/35095?open=rating)

**Comment by dmarkov** [comment]
@rultor deploy pls

**Pull request merged**